### PR TITLE
Add Charbeat config values for Feature Index page 

### LINKED ISSUE #7910: Define Chartbeat config values specific to the FIX page
I think we might need to confirm what the values for those would be in the FIX pages data and set them appropriately in [this file](https://github.com/bbc/simorgh/blob/latest/src/app/containers/ChartbeatAnalytics/utils/index.js#L104-L125) for `getType` and `getTitle`

_Originally posted by @rhenshaw56 in https://github.com/bbc/simorgh/pull/7856#discussion_r494270309_

### PR DESCRIPTION
**Pull request by chewu**
Resolves #7910

**Overall change:**
We are using the same values as the Index page as instructed by @JonBeeb.

**Code changes:**

- _A bullet point list of key code changes that have been made._
- _When describing code changes, try to communicate **how** and **why** you implemented something a specific way, not just **what** has changed._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by karinathomasbbc — APPROVED** [review]
Looks good, thanks!

File: src/app/containers/ChartbeatAnalytics/utils/index.test.js [hunk 1]
```diff
@@ -784,6 +790,86 @@ describe('Chartbeat utilities', () => {
...
+    };
+
+    const expectedConfig = {
+      domain: 'test.bbc.co.uk',
+      idSync: {
+        bbc_hid: 'foobar',
+      },
+      path: '/',
+      sections: 'Afrique, Afrique - FIX',
+      title: 'This is a Feautre Index page title',
```
Comment (karinathomasbbc) [inline src/app/containers/ChartbeatAnalytics/utils/index.test.js]: Typo? `This is a Feature Index page title`

**Review by karinathomasbbc — APPROVED** [review]

File: src/app/containers/ChartbeatAnalytics/utils/index.test.js (hunk 1 above)
Comment (chewu) [reply src/app/containers/ChartbeatAnalytics/utils/index.test.js]: DOH

**Review by chris-hinds — APPROVED** [review]

**Comment by chris-hinds** [comment]
Changes look good to me. Looks like you need to update the tests to reflect the Fix title change.

**Review by simonsinclair — APPROVED** [review]
This looks great, Che. 👍

**Pull request merged**